packages/stark-ui/src/modules/collapsible/components/_collapsible-theme.scss

```diff
@@ -0,0 +1,44 @@
+/* ============================================================================== */
+/*  S t a r k   C o l l a p s i b l e   -   T h e m e                             */
+/* ============================================================================== */
+/* stark-ui: src/modules/collapsible/components/_collapsible-theme.scss */
+
+.stark-collapsible {
+  .mat-expansion-panel {
+	box-shadow: $elevation-1;
+	&.mat-expanded {
+	  box-shadow: $elevation-16;
+	}
+
+	.mat-expansion-panel-header {
+	  border-bottom: 1px solid map-get($mat-light-theme-background, disabled-button);
+	  background-color: map-get($mat-light-theme-background, background-light);
+	  &:not(.mat-expanded) {
+		&:hover {
+		  background-color: map-get($mat-light-theme-background, background-light);
+		}
+	  }
+	  &.mat-expanded {
+		background-color: map-get($mat-light-theme-background, card);
+	  }
+	}
+
+	.mat-expansion-panel-body {
+	  background-color: map-get($mat-light-theme-background, background-light);
+	}
+
+	.mat-expansion-panel-header-description {
+	  color: mat-color(map-get($base-theme, primary-palette), 900);;
+	}
+
+	.mat-expansion-panel-header-title {
+	  color: mat-color(map-get($base-theme, primary-palette), 500);
+	}
+
+	.stark-collapsible-icon {
```

Comment:
Now that the PR #678 has been merged, the logic to set the color to our components changed. You can have a look on how it is done in the Dropdown component ;)

You can also ask @catlabs about the implementation details...